homework/week-0/AllisonBojarski/TextAdvGame(HWproblem#3)/src/nyc/c4q/alizinha/Enemy.java

```diff
@@ -0,0 +1,18 @@
+package nyc.c4q.alizinha;
+/**
+ * Created by c4q-Allison on 5/6/15.
+ */
+public class Enemy extends Character implements Defender
+{
+    public Enemy(String charName, int healthPoints, Weapon weapon, Room room)
+    {
+        super(charName, healthPoints, weapon, room);
+    }
+
+    @Override
+    public int calcShieldSize(int damagePoints)
+    {
+        return 10;
```

Comment:
This value is never really used. A suggestion for this would be to reduce the incoming damage during a battle by this amount if the character is holding a shield.